jetty-server/src/main/config/etc/jetty.xml

```diff
@@ -105,7 +105,7 @@
     <!-- =========================================================== -->
     <!-- extra server options                                        -->
     <!-- =========================================================== -->
-    <Set name="stopAtShutdown" property="jetty.server.stopAtShutdown"/>
+    <Set name="stopAtShutdown"><Property name="jetty.server.stopAtShutdown" default="true"/></Set>
```

Comment:
Does not this require a test with the `JettyHomeTester`?

Reply:
I tried, but it's just not possible with the current setup. When `destroy() `is called on the remote jetty process the IO pipes to that process seem to be immediately closed, so we never see the appropriate `"Stopped Server"` log message (although it is generated, as running the distro at the command line attests). I think this is a much bigger issue, namely how do we test each and every module to ensure correctness, so I've opened a separate issue: #6373